### PR TITLE
Default plugins: i18n for the plugins categories

### PR DESCRIPTION
**Pull request by yoavf**
Add missing i18n

Test live: https://calypso.live/?branch=fix/wpcom-plugins-i18n

### CODE REVIEW

**Comment by tyxla** [comment]
LGTM 🚢

File: client/my-sites/plugins-wpcom/default-plugins.js [hunk 1]
```diff
@@ -8,147 +8,147 @@ export const defaultStandardPlugins = [
 		name: i18n.translate( 'Stats' ),
 		descriptionLink: 'https://support.wordpress.com/stats/',
 		icon: 'stats-alt',
-		category: 'Traffic Growth',
+		category: i18n.translate( 'Traffic Growth' ),
```
Comment (retrofox) [inline client/my-sites/plugins-wpcom/default-plugins.js]: I see that the same string is translated five times. Maybe it worth make a definitions object and thus call the translate function as little as possible.

```es6
const translations = {
  trafficGrowth: i18n.translate( 'Traffic Growth' ),
  appearance: i18n.translate( 'Appearance' ),
  // ...
};
```

then ...

```es6
  category: translations[ trafficGrowth ],
```

```es6
  category: translations[ appearance ],
```

In this way each string is translated once. You could implement it in a different way though.
My two coins. :-D

**Comment by lancewillett** [comment]
@yoavf Waiting for reply still.

**Comment by lancewillett** [comment]
Ping @yoavf again.

**Comment by yoavf** [comment]
Was already taken care of elsewhere

**Pull request closed without merging**